source/app/views/layouts/_home-navbar.html.erb

```diff
@@ -0,0 +1,38 @@
+<!-- This is a sample header just to get a feel to the website, frontend dev pls make it look better :) -->
+<nav class="navbar navbar-expand-lg navbar-light bg-white">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="<%= root_path %>">
+      <%= image_tag 'real ratings.svg', alt: 'Real Rating Logo', class: 'd-inline-block align-text-top', size: '200x40' %>
+    </a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse justify-content-between" id="navbarNav">
+      <ul class="navbar-nav mx-auto">
+        <li class="nav-item">
+          <a class="nav-link <%= 'active' if selected == 'home' %>" href="<%= root_path %>">Home</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link <%= 'active' if selected == 'about' %>" href="<%= about_path %>">About</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link <%= 'active' if selected == 'contact' %>" href="<%= contact_path %>">Contact</a>
+        </li>
+      </ul>
+      <ul class="navbar-nav me-4">
+        <% if user_signed_in? %>
+          <li class="nav-item">
+            <%= link_to 'Log Out', destroy_user_session_path, method: :delete, class: 'btn btn-dark ms-2' %>
```

Comment:
this should be: 
`<%= link_to 'Log Out', destroy_user_session_path, method: :delete, data: { turbo_method: :delete }, class: 'btn btn-dark ms-2' %>`